docs/documentation/server_admin/topics/oid4vci/vc-issuer-configuration.adoc

```diff
@@ -0,0 +1,361 @@
+== Configuring Keycloak as a Verifiable Credential Issuer
+
+{project_name} provides support for https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html[OpenID for Verifiable Credential Issuance].
```

Comment:
It can be good to start the documentation with the note that this is experimental feature and should be never used in the production and that it is not guaranteed that be backwards compatible. See some other documentation sections for the inspiration, which use the similar note for the "Tech preview" features.